### PR TITLE
fix(cc-block): fix css and description text

### PR DESCRIPTION
**Pull request by HeleneAmouzou**
## What does this PR do?

- `cc-env-var-form` : adds plural to "variable" in description text and removes unnecessary "En savoir plus / Learn more" in description text,
- `cc-block` : removes margin bottom,
- `cc-domain-management` : reverts to initial gap,
- `cc-invoice`: remove unnecessary padding.

## How to review?

- Check the commit,
- Check the components in [preview](https://clever-components-preview.cellar-c2.services.clever-cloud.com/cc-block/fix-component/index.html)

### CODE REVIEW

**Review by florian-sanders-cc — CHANGES_REQUESTED** [review]
Only a small nitpick, you have a typo in the first commit desc:

> fix(cc-env-var-form): remove unnecessary link and fix type in description text
should be:

> fix(cc-env-var-form): remove unnecessary link and fix typo in description text

Apart from that it's great, gj :wink:

File: src/translations/translations.en.js [hunk 1]
```diff
@@ -437,11 +437,11 @@ export const translations = {
   //#endregion
   //#region cc-env-var-form
   'cc-env-var-form.description.config-provider': /** @param {{addonName: string}} _ */ ({ addonName }) =>
-    sanitize`Configuration exposed to dependent applications. <a href="https://www.clever-cloud.com/doc/deploy/addon/config-provider/">Learn more</a><br>These variables will be injected as environment variables in applications that have the add-on <strong>${addonName}</strong> in their service dependencies.<br>Every time you update your changes, all the dependent applications will be automatically restarted.`,
+    sanitize`Configuration exposed to dependent applications.<br>These variables will be injected as environment variables in applications that have the add-on <strong>${addonName}</strong> in their service dependencies.<br>Every time you update your changes, all the dependent applications will be automatically restarted.`,
```
Comment (hsablonniere) [inline src/translations/translations.en.js:440]: issue: this learn more is not the same link as in the footer, I think we should keep it

File: src/translations/translations.en.js [hunk 2]
```diff
@@ -437,11 +437,11 @@ export const translations = {
...
   'cc-env-var-form.description.env-var': /** @param {{appName: string}} _ */ ({ appName }) =>
-    sanitize`These variables will be injected as environment variables in the application <strong>${appName}</strong>. <a href="https://developers.clever-cloud.com/doc/reference/reference-environment-variables/">Learn more</a>`,
+    sanitize`These variables will be injected as environment variables in the application <strong>${appName}</strong>.`,
```
Comment (hsablonniere) [inline src/translations/translations.en.js:442]: praise: this one should be remove, this is OK :+1:

File: src/translations/translations.fr.js [hunk 3]
```diff
@@ -446,12 +446,12 @@ export const translations = {
   //#endregion
   //#region cc-env-var-form
   'cc-env-var-form.description.config-provider': /** @param {{addonName: string}} _ */ ({ addonName }) =>
-    sanitize`Configuration publiée pour les applications dépendantes. <a href="https://www.clever-cloud.com/doc/deploy/addon/config-provider/">En savoir plus</a><br>Ces seront injectées en tant que variables d'environnement dans les applications qui ont l'add-on <strong>${addonName}</strong> dans leurs services liés.<br>À chaque fois que vous mettez à jour les changements, toutes les applications dépendantes seront redémarrées automatiquement.`,
+    sanitize`Configuration publiée pour les applications dépendantes.<br>Ces seront injectées en tant que variables d'environnement dans les applications qui ont l'add-on <strong>${addonName}</strong> dans leurs services liés.<br>À chaque fois que vous mettez à jour les changements, toutes les applications dépendantes seront redémarrées automatiquement.`,
```
Comment (hsablonniere) [inline src/translations/translations.fr.js:449]: issue: this learn more is not the same link as in the footer, I think we should keep it

File: src/translations/translations.fr.js [hunk 4]
```diff
@@ -446,12 +446,12 @@ export const translations = {
...
   'cc-env-var-form.description.env-var': /** @param {{appName: string}} _ */ ({ appName }) =>
-    sanitize`Ces variables seront injectées en tant que variables d'environnement dans l'application <strong>${appName}</strong>. <a href="https://developers.clever-cloud.com/doc/reference/reference-environment-variables/">En savoir plus</a>`,
+    sanitize`Ces variables seront injectées en tant que variables d'environnement dans l'application <strong>${appName}</strong>.`,
   'cc-env-var-form.description.exposed-config': /** @param {{appName: string}} _ */ ({ appName }) =>
-    sanitize`Configuration publiée pour les applications dépendantes. <a href="https://www.clever-cloud.com/doc/admin-console/service-dependencies/">En savoir plus</a><br>Ces variables ne seront pas injectées dans l'application <strong>${appName}</strong>, elles seront injectées en tant que variables d'environnement dans les applications qui ont <strong>${appName}</strong> dans leurs services liés.`,
```
Comment (hsablonniere) [inline src/translations/translations.fr.js:453]: issue: this learn more is not the same link as in the footer, I think we should keep it

**Review by hsablonniere — CHANGES_REQUESTED** [review]
Well done @HeleneAmouzou, I think we should keep the `Learn more` links for "exposed-config" and "config-provider" as they're not the same as the link in the footer (which is also useful).

**Pull request merged**